tests/integration-templates/package.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
imo I would make that part of the workspace to avoid multiplication of package-lock.json and complexifying dependency upgrade

Reply:
Updated